aspen/algorithms/website.py

```diff
@@ -146,9 +147,9 @@ def delegate_error_to_simplate(website, state, response, request=None, resource=
 
     if fspath is not None:
         request.original_resource = resource
-        if resource is not None and resource.is_bound:
+        if resource is not None and resource.default_media_type != website.media_type_default:
```

Comment:
a) maybe, but `website` is a semi-public API so I was trying not to change it too much.
b) read that as "if there's a resource with a default media type different than the global default media type".  It's a check for approximately what `is_bound` used to mean: it's got a media type associated with it by default (probably because it has an extension like foo.html.spt)